CODEOWNERS

```
@@ -1 +1 @@
-*       @momack2 @arajasek @jennijuju @kaitlin-beegle @anorth @raulk @jsoares
+*       @momack2 @arajasek @jennijuju @kaitlin-beegle @anorth @raulk @jsoare @TippyFlitsUK
```

```suggestion
*       @momack2 @arajasek @jennijuju @kaitlin-beegle @anorth @raulk @jsoares @TippyFlitsUK
```

I assume this was an accidental typo, but Jorge's handle is still @jsoares with the "s"

---

thx @kaitlin-beegle :)